packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -244,14 +235,28 @@ class LocalStoreTester {
     return this;
   }
 
+  toNotContainIfEager(
+    gcIsEager: boolean,
+    doc: MaybeDocument
+  ): LocalStoreTester {
+    if (gcIsEager) {
```

Comment:
Can we pass `gcIsEager` to LocalStoreTester? That way, the tests would not need to know what GC mode they are running under.